artiq/coredevice/i2c.py

```diff
@@ -157,12 +164,11 @@ def select(self, mask):
 
         :param mask: Bit mask of enabled channels
         """
-        i2c_write_byte(self.busno, self.address, mask)
+        i2c_pca954x_select(self.busno, self.address, mask)
```

Comment:
You could just remove the ``select`` function, only keep ``set``, and add ``unset``.